src/cli/actions.js

```diff
@@ -540,7 +550,28 @@ function writeTemplate(id, type, output, content, templates, huron) {
 
   templates.set(key, `./${outputRelative}`, storeCb);
   fs.outputFileSync(outputPath, content);
-  console.log(`writing ${outputPath}`);
+  console.log(`Writing ${outputPath}`);
+}
+
+/**
+ * Copy an HTML file into the huron output directory.
+ * @param  {string} filename - The name of the file (without extension).
+ * @param  {string} output - The relative path to this file within the huron.kss directory.
+ * @param  {string} content - The content of the file to write.
+ * @param  {object} huron - The huron config object.
+ * @see writeTemplate()
+ * @todo We can look to this function to load the handlebars templates
+ *       through the webpack loader instead.
+ */
+function copyFile(filename, output, content, huron) {
+  const outputRelative = path.join(huron.templates, output, `${filename}.html`);
+  const outputPath = path.resolve(cwd, huron.root, outputRelative);
+  try {
+    fs.outputFileSync(outputPath, content);
```

Comment:
If you're feeling ambitious, you could start switching these sync file outputs to async (`fs.outputFile`) and use a promise.